docs/use/setup-wallet/index.md

```diff
@@ -48,7 +48,7 @@ On Ethereum, the system supports [MetaMask](https://metamask.io/download.html),
 
 ![](./images/wallet-component-eth.png#width=40%;)
```

Comment:
Line 49:
![](./images/wallet-component-eth.png#width=40%;)
Change to 
![image](https://github.com/centrifuge/documentation/assets/90972248/9fb21bb5-2b84-45ea-a8bc-2214f13efb63)